client/lib/analytics/ad-tracking.js

```diff
@@ -197,10 +197,9 @@ if ( typeof window !== 'undefined' ) {
  */
 function maybeRefreshCountryCodeCookieGdpr( callback ) {
 	const cookieMaxAgeSeconds = 6 * 60 * 60;
-
 	const cookies = cookie.parse( document.cookie );
 
-	if ( ! cookies.country_code || 'unknown ' === cookies.country_code ) {
+	if ( ! cookies.country_code ) {
```

Comment:
I see, so when a GEOIP request fails (which is when we set it to `unknown`) instead of requesting it all the time we request it only once every 6 hours as usual, is that the intent?

Reply:
Exactly. It seems that is the strategy we have been using here already (due to the space). So I don't think this really was a bug. More like an accidental feature.

I think if we were checking again on every failure, then many of the ad trackers would end up firing the HTTP request to the API over and over again anyway; i.e., giving the way this method is called upon by others.